app/src/app/api/v0/auth/requestpassword/route.ts

```diff
@@ -0,0 +1,36 @@
+import { db } from "@/models";
+import { apiHandler } from "@/util/api";
+import { passwordRegex, resetPasswordRequest } from "@/util/validation";
+import bcrypt from "bcrypt";
+import createHttpError from "http-errors";
+import jwt from "jsonwebtoken";
+import { Session } from "next-auth";
+import { NextRequest, NextResponse } from "next/server";
+import { z } from "zod";
+
+const requestPassword = z.object({
+  password: z.string().min(4).regex(passwordRegex),
+});
+
+export const POST = apiHandler(
+  async (
+    _req: NextRequest,
```

Comment:
If you are using a function argument, name it without the _
```suggestion
    req: NextRequest,
```

Initial _ is reserved to mark function arguments as unused.